_includes/footer.html

```diff
@@ -1,3 +1,4 @@
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/rellax/1.0.0/rellax.min.js"></script>
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
 <script type="text/javascript" src="{{ '/assets/js/app.js' | prepend: site.baseurl }}"></script>
+<script id="dsq-count-scr" src="//hackdown.disqus.com/count.js" async></script>
```

Comment:
There is an if statement above checking for comments, so shouldn't there be an if statement checking if this script should even be loaded?

Reply:
I'll fix all this and let you know, thanks for the feeback!

Reply:
Removed disqus comments, consider this fixed.